AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AirlineReservation.Models.Data;
+using AirlineReservation.Services.Airline;
+using AirlineReservation.Controllers;
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+
+namespace AirlineReservation.Tests.Test.Controller
+{
+    public class AirlineControllerTest
+    {
+        private readonly Fixture _fixture = new Fixture();
+
+        [Fact]
+        public async Task ShouldReturnOn_GetAllAirlines_AndStatusAs200()
+        {
+            var mockAirlinesService = new Mock<IAirlineService>();
+            var mockAirlines = _fixture.Create<List<AirlineModel>>();
+            mockAirlinesService.Setup(service => service.getAllAirlines()).ReturnsAsync(mockAirlines);
+
+            var airlineController = new AirlineController(mockAirlinesService.Object);
+            var result = await airlineController.GetAllAirlines();
+
+            mockAirlinesService.Verify(service => service.getAllAirlines(), Times.Once);
+
+            result
+                .Should().BeOfType<OkObjectResult>().Which.Value.Should()
+                .BeOfType<List<AirlineModel>>().And.Subject.As<List<AirlineModel>>()
+                .Should().BeEquivalentTo(mockAirlines);
+
+            result
+                .Should().BeOfType<OkObjectResult>().Which.StatusCode.Should()
+                .Be(StatusCodes.Status200OK);
```

Comment:
OkObjectResult by definition has a status code of 200, This feels redundant

Reply:
Checked with OkObjectResult.